element_handle.go

```diff
@@ -285,3 +303,13 @@ func normalizeFilePayloads(files []InputFile) []map[string]string {
 	}
 	return out
 }
+
+func transformToStringList(in interface{}) ([]string, error) {
```

Comment:
This function can never return an error, so I would also adjust the function signature corresponding to that.